doc/fluid/howto/optimization/timeline.md

```diff
@@ -0,0 +1,27 @@
+## how to use timeline tool to do profile
+
+1. Add `with profiler.profiler(...)` to the main training loop. After run, the code will generate a profile record file `/tmp/profile`.
+
+	```python
+	with profiler.profiler('All', 'total', '/tmp/profile') as prof:
+	    for pass_id in range(pass_num):
```

Comment:
Add a warning here:
Don't run many steps (2~3), otherwise the profile will be too large. The profile is only generated after it exits the scope